W/wxWidgets/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+using BinaryBuilder, Pkg
+
+name = "wxWidgets"
+version = v"3.1.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/wxWidgets/wxWidgets/releases/download/v$(version)/wxWidgets-$(version).tar.bz2",
+                  "3ca3a19a14b407d0cdda507a7930c2e84ae1c8e74f946e0144d2fa7d881f1a94")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/wxWidgets-*/
+if [[ "${target}" == *-linux-musl ]]; then
+    # Delete libexpat to prevent it from being picked up by mistake
+    rm /usr/lib/libexpat.so*
+fi
+export LDFLAGS="-liconv"
```

Comment:
I'm not sure forcing linking everything to iconv when only a single library only on FreeBSD misses it is a good idea.  Definitely not good to do for all platforms.  I'd rather like to understand why the build system misses to automatically inject this flag for FreeBSD (or, what does it do on the other platforms?)

Reply:
Why'd the conditional statement not work?

Reply:
I see there are some dozens of different libraries that get compiled, not all of them need to link to iconv and there is no reason to force the link for no good reason.  It's different when there is a single library in output, in that case forcing `LDFLAGS=-liconv` would make more sense

Reply:
I'm not 100% sure if this is the correct explanation but

freebsd provides its own libiconv, which might have some things missing, hence causing the error
I hear we can use the one provided by gnu by using ``` -Diconv=external ```

Reply:
Another cause might be the fact the iconv by freebsd doesn't support something called translit
A workaround for that might be USES=iconv:translit....

Reply:
Giving the first solution is worth a try
Since I don't know how to do that, can you commit the proper changes？（It'll serve as a good reference for me for next time）